### PR TITLE
[4.2] content query comments

### PR DESCRIPTION
**Pull request by brianteeman**
Fix copy paste etc errors in the comment description of the functions.

Code review-

### CODE REVIEW

File: components/com_content/src/Helper/QueryHelper.php [hunk 1]
```diff
@@ -157,7 +157,7 @@ public static function orderbySecondary($orderby, $orderDate = 'created', Databa
     }
 
     /**
-     * Translate an order code to a field for primary category ordering.
+     * Translate an order code to a field for date ordering.
```
Comment (sandewt) [inline components/com_content/src/Helper/QueryHelper.php:160]: status instead of date ordering ?

Comment (brianteeman) [reply components/com_content/src/Helper/QueryHelper.php:160]: why?

Comment (sandewt) [reply components/com_content/src/Helper/QueryHelper.php:160]: modified, published, unpublished and created show the status

Comment (brianteeman) [reply components/com_content/src/Helper/QueryHelper.php:160]: image

Comment (sandewt) [reply components/com_content/src/Helper/QueryHelper.php:160]: issue

Comment (brianteeman) [reply components/com_content/src/Helper/QueryHelper.php:160]: thats unrelated

Comment (sandewt) [reply components/com_content/src/Helper/QueryHelper.php:160]: I see

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on 1ed3d8ec7217bdc8947d5d1bc34f88c5b63760d7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38253">issues.joomla.org/tracker/joomla-cms/38253</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 1ed3d8ec7217bdc8947d5d1bc34f88c5b63760d7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38253">issues.joomla.org/tracker/joomla-cms/38253</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38253">issues.joomla.org/tracker/joomla-cms/38253</a>.</sub>

**Comment by roland-d** [comment]
Thank you

**Pull request merged**